packages/botonic-core/src/models/events/message/element.ts

```diff
@@ -0,0 +1,7 @@
+import { WithButtons } from './buttons'
+
+export interface CarouselElement extends WithButtons {
+  src: string
+  title: string
+  subtitle?: string
+}
```

Comment:
Is the image missing here, right?

Reply:
well, I did rename it as `src` instead of `pic`, here the confusion maybe, as this is how I named it in the parser. Maybe `pic` is more explicit given the context . Renaming it